packages/invoices/src/InboundShipment/DetailView/api.ts

```diff
@@ -327,7 +338,7 @@ export const useInboundFields = <KeyOfInvoice extends keyof Invoice>(
     }
   );
 
-  const update = useDebounceCallback(mutateAsync, [], timeout);
+  const update = useDebounceCallback(mutate, [], timeout);
```

Comment:
The `mutate` fn allows for passing extra options from the caller, which is nice to be able to add toasts etc